testbed/coav-sim.sh

```diff
@@ -79,8 +104,8 @@ run_gazebo () {
     # Gazebo engine without GUI.
     # The log can be played through `gazebo -p logfile`
     SDFFILE="${SCRIPT_DIR}/worlds/${WORLD}"
-    gzserver --record_path $LOGDIR --verbose $SDFFILE \
-        > ${LOGDIR}/gzserver.log 2> ${LOGDIR}/gzserver_err.log &
+    GZCMD="gzserver --record_path $LOGDIR --SILENT $SDFFILE"
```

Comment:
I didn't get the '--SILENT', what is the intention? gzserver --help doesnt list this arg and issuing it by hand with or without it doesn't change my output. Without verbose it miss some warning that right now aren't relevant but it may be if you screw something up.